packages/astro/src/core/endpoint/index.ts

```diff
@@ -51,6 +52,6 @@ export async function call(
 
 	return {
 		type: 'simple',
-		body: response.body,
+		...response,
```

Comment:
would prefer setting the properties instead of a spread.

Reply:
Because of typescript we’re not at risk of spreading more than we intended. It is safe. And if people add more endpoint properties they come for free.

Would you still prefer explicit over implicit? If so I can make the change

Reply:
Yeah, prefer explicit to keep the code readable. Thanks!